babel.config.js

```diff
@@ -4,11 +4,21 @@ module.exports = (api) => {
     ['@babel/plugin-transform-react-jsx', {
       runtime: 'automatic',
       importSource: 'preact',
+      useBuiltIns: true
     }],
     //['@babel/plugin-transform-runtime', {useESModules: true}],
   ];
   const presets = [
-    '@babel/preset-env',
+    [
+      '@babel/preset-env',
+      {
+        loose: true,
+        bugfixes: true,
+        targets: {
+          esmodules: true
+        }
```

Comment:
There seems to be no effect on the `App` function generated code if `bugfixes`+`targets` are included or not?